packages/modeling/src/geometries/poly2/arePointsInside.test.js

```diff
@@ -16,7 +16,7 @@ test('poly2: arePointsInside() should return proper values', (t) => {
   obs = arePointsInside([[0, 0]], polygon)
   t.is(obs, 0)
   obs = arePointsInside([[5, 0]], polygon)
-  t.is(obs, 0)
+  // t.is(obs, 0) // FAILS
```

Comment:
What's going on with these?  Do they fail for good reason?  Should they be removed totally? Should the code above that supports them be removed?

Reply:
It’s kind of complicated... but basically boils down to ‘is an edged inside or outside’?

http://geomalgorithms.com/a03-_inclusion.html

Because the crossing test is being used, the far outside edges are considered ‘outside’.

I really want to use the winding test, but it fails. 😔